docs/DeveloperGuide.adoc

```diff
@@ -729,7 +729,65 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
-|`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
+|`* * *` |user |add a new location |store important locations and establishments
+
+|`* * *` |user |filter contacts by tags |view specific contacts that are assigned with specific tags
+
+|`* * *` |user |the browser to automatically search for the address of the selected person in Google Maps |
+
+|`* * *` |user who travels a lot |to know the shortest distance from one contact's address to another |
+
+|`* * *` |user who travels a lot |set a reference location |find the shortest distance from my reference location to a contact's location
+
+|`* *` |user |add profile pictures to contacts |
+
+|`* *` |user |have a default picture for contacts without profile pictures |know which contact does not have a profile picture
+
+|`* *` |user |update the profile pictures of contacts |
+
+|`* *` |new user |input instructions into a chatbot interface |I do not need to remember the format of commands
+
+|`* *` |new user |to have an autocomplete for the commands |I do not need to remember the format of commands
+
+|`* *` |secretive user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
+
+|`* *` |lazy user |filter contacts by distance from my reference location |see a list of contacts that are at a specific radial distance from me
+
+|`* *` |user |store my work and home addresses |use these addresses as destinations/sources of my journeys
+
+|`* *` |lazy user |send and receive contact details to and from my friends |minimize the amount of data inputs
+
+|`* *` |careful user |specify the type contact details I send to friends |protect my privacy
+
+|`* *` |user |import profile pictures of contacts from their social media |upload profile pictures of contacts easily
+
+|`* *` |user |have the option of adding a birthday to a contact |store the addresses of my contacts
+
+|`* *` |forgetful user |be reminded of a person's birthday (if valid) |in case I forget the date
+
+|`* *` |thoughtful user |view the birthdays of my contacts in sorted order |know whose birthday is coming up
+
+|`* *` |user involved in many group projects or activities |create telegram group chats of contacts tagged with the same tag |add contacts into a group chat easily
```

Comment:
Perhaps this could be `*` as it would be challenging and the addressbook can still function without this feature.